products/sle/main.pm

```diff
@@ -1001,31 +1001,61 @@ sub load_slenkins_tests {
 
 sub load_ha_cluster_tests {
     return unless (get_var("HA_CLUSTER"));
+    loadtest "boot/boot_to_desktop";
     loadtest "ha/wait_support_server";
-    loadtest "installation/first_boot";
```

Comment:
Hm, if you see that boot _to_desktop can replace first_boot, maybe we should rework first_boot to use more common code?

Reply:
This change was suggested to me by @czerw.
It works like the old behavior. So yes maybe it can be use as a replacement or some part could be shared.

If you want, I can have a look at this but later, I think time is to short for this PR.

Reply:
agreed

Reply:
OK, I've created [poo#27904](https://progress.opensuse.org/issues/27904) for this.